ingest/bin/post_process_metadata.py

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/env python3
+
+"""Reformat pandas DataTables for a pathogen build.
+
+Expecting one argument, a NCBI Virus metadata.tsv file
+"""
+# ===== Dependencies
+import argparse
+import os
+import sys
+
+import numpy as np
+import pandas as pd
+
+
+def parse_args():
+    # Main help command
+    parser = argparse.ArgumentParser(
+        description="Reformat a NCBI Virus metadata.tsv file for a pathogen build."
+    )
+    # Add first argument
+    parser.add_argument(
+        "--metadata", help="NCBI Virus metadata.tsv file.", required=True
+    )
+    parser.add_argument(
+        "--outfile",
+        help="Output file name, e.g. processed_metadata.tsv.",
+        required=True,
+    )
+
+    return parser.parse_args()
+
+
+# === Private methods
+
+
+def _set_strain_name(record):
+    """Check Isolate_s and Strain_s to find the strain name"""
+    if record["strain"] != record["accession"]:
+        strain_name = record["strain"]
+    elif record["strain"] == record["accession"] and pd.notna(record["strain_s"]):
+        strain_name = record["strain_s"]
+    else:
+        strain_name = record["strain"]
+
+    return (
+        strain_name.replace(" ", "_")
+        .replace("-", "_")
+        .replace(".", "_")
+        .replace("(", "_")
+        .replace(")", "_")
+    )
+
+
+def _set_url(record):
+    """Set url column from accession"""
+    return "https://www.ncbi.nlm.nih.gov/nuccore/" + str(record["accession"])
+
+
+def _set_paper_url(record):
+    """Set paper_url from publication"""
+    if pd.notna(record["publications"]):
+        paper_url = "https://www.ncbi.nlm.nih.gov/pubmed/" + str(record["publications"])
+        return paper_url.split(",")[0]
```

Comment:
It is very weird to me to stringify a list, append that to a URL, and _then_ split on commas. Why not something like the following instead?

```suggestion
        return "https://www.ncbi.nlm.nih.gov/pubmed/" + str(record["publications"][0])
```

Reply:
Resolved with https://github.com/nextstrain/dengue/pull/6/commits/5dd06fd05725903920b863c4ff46d31a4edce2a4